src/CMakeLists.txt

```diff
@@ -30,24 +30,23 @@
 # Boston, MA 02111-1307, USA.
 # =============================================================================
 
-
-
-add_library("${LIBRASCAL_NAME}" "")
-
-set(rascal_SRC
-  ${CMAKE_CURRENT_SOURCE_DIR}/json_io.cc
-  ${CMAKE_CURRENT_SOURCE_DIR}/units.cc
-  ${CMAKE_CURRENT_SOURCE_DIR}/rascal_utility.cc
-  )
-target_sources("${LIBRASCAL_NAME}" PRIVATE ${rascal_SRC})
-
-
-target_link_libraries("${LIBRASCAL_NAME}" PUBLIC Eigen3::Eigen)
-target_link_libraries("${LIBRASCAL_NAME}" PUBLIC "${WIGXJPF_NAME}")
-
-add_subdirectory(structure_managers)
-add_subdirectory(representations)
-add_subdirectory(utils)
-add_subdirectory(math)
-
-install(TARGETS "${LIBRASCAL_NAME}" DESTINATION lib)
+set(rascal_sources
+    json_io.cc
```

Comment:
Is this convention to set `.cc` files by hand?

Reply:
This is how it was already done in the code.

The alternative is to use `file(GLOB ...)`, but this mean that we have to manually re-run cmake when adding/deleting files.